src/settings/target/site.rs

```diff
@@ -1,14 +1,18 @@
+use std::env;
+use std::fs;
 use std::path::PathBuf;
 
 use serde::{Deserialize, Serialize};
 
+use crate::commands::generate::run_generate;
+
 const SITE_ENTRY_POINT: &str = "workers-site";
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct Site {
     pub bucket: String,
     #[serde(rename = "entry-point")]
-    pub entry_point: Option<String>,
+    entry_point: Option<PathBuf>,
```

Comment:
this smells like you could fix up `bucket` while you're at https://github.com/cloudflare/wrangler/issues/622

Reply:
(but you don't have to)

Reply:
DID IT